src/test/java/io/appium/java_client/ios/IOSDriverTest.java

```diff
@@ -34,13 +35,22 @@
 import org.openqa.selenium.By;
 import org.openqa.selenium.ScreenOrientation;
 import org.openqa.selenium.html5.Location;
+import org.openqa.selenium.remote.Response;
+import org.openqa.selenium.remote.http.HttpMethod;
 import org.openqa.selenium.support.ui.ExpectedConditions;
 import org.openqa.selenium.support.ui.WebDriverWait;
 
 import java.time.Duration;
 
 public class IOSDriverTest extends AppIOSTest {
 
+    @Test
+    public void addCustomCommandTest() {
+        driver.addCommand(HttpMethod.GET, "/sessions", "getSessions");
```

Comment:
Do you have examples of placeholders?
e.g. session id, element id etc
https://github.com/appium/ruby_lib_core/blob/0f93a52bbdc44bf916c9b974fe9fd09d48e5ff39/lib/appium_lib_core/common/base/driver.rb#L107-L145
https://github.com/appium/python-client/blob/e48085d27968a93d9cfb5d86964ac84198b52214/test/unit/webdriver/webdriver_test.py#L285-L309

Reply:
Sure @KazuCocoa will add it.